pgxn/neon/pagestore_client.h

```diff
@@ -28,10 +28,17 @@
 #define MAX_SHARDS 128
 #define MAX_PAGESERVER_CONNSTRING_SIZE 256
 
+/*
+ * Right now protocal version is not set to the server.
+ * So it is ciritical that format of existed commands is not changed.
+ * New protocl versions can just add new commands.
+ */
+#define NEON_PROTOCOL_VERSION  2
+
 typedef enum
 {
 	/* pagestore_client -> pagestore */
-	T_NeonExistsRequest = 0,
+	T_NeonExistsRequest = 10, /* new protocol message tags start from 10 */
```

Comment:
Wasn't obvious to me that this field encodes the protocol version. It would be nicer to somehow include `NEON_PROTOCOL` version in the right hand side of the assignment and make it clearer.

Reply:
I wrote the comment explaining that we do not have protocol version now and it is not possible to introduce it without breaking backward compatibility. This is why the requirement is that any changes of protocol require introducing of new command tags, preserving format of old commands unchanged.

I agree with you that explicit passing of protocol version will be better, but relying on commands tag also seems to be acceptable. At least I hope we will not change this protocol too frequently...

Reply:
Now the whole compute<>pageserver communications start with `pagestream` keyword. We can start it with e.g. `pagestream_v2` to make new protocol version to be explicit.

Reply:
It is actually done in @hlinnaka 's version of this PR: #7377